syntax = "proto2";

package fyModuleBackInfo; //后模块信息

message ModuleBackFrame
{
	required float output_current_01 = 1;     //制动灯
	required float output_current_02 = 2;     //备用
	required float output_current_03 = 3;     //后雾灯
	required float output_current_04 = 4;     //备用
	required float output_current_05 = 5;     //左转向灯
	
	required float output_current_06 = 6;     //备用
	required float output_current_07 = 7;     //右转向灯
	required float output_current_08 = 8;     //干燥器电源
	required float output_current_09 = 9;     //倒车灯
	required float output_current_10 = 10;    //备用

	required float output_current_11 = 11;    //前门踏步灯
	required float output_current_12 = 12;    //位置灯
	required float output_current_13 = 13;    //中门踏步灯
	required float output_current_14 = 14;    //备用
	required float output_current_15 = 15;    //备用

	required bool  switch_01 = 16;            //制动灯
	required bool  switch_02 = 17;            //备用
	required bool  switch_03 = 18;            //后雾灯
	required bool  switch_04 = 19;            //备用
	required bool  switch_05 = 20;            //左转向灯
	required bool  switch_06 = 21;            //备用
	required bool  switch_07 = 22;            //右转向灯
	required bool  switch_08 = 23;            //干燥器电源
	required bool  switch_09 = 24;            //倒车灯
	required bool  switch_10 = 25;            //备用
	required bool  switch_11 = 26;            //前门踏步灯
	required bool  switch_12 = 27;            //位置灯
	required bool  switch_13 = 28;            //中门踏步灯
	required bool  switch_14 = 29;            //备用
	required bool  switch_15 = 30;            //备用
	required bool  switch_16 = 31;            //水位传感器开关
	
	required bool  switch_17 = 32;            //备用
	required bool  switch_18 = 33;            //备用
	required bool  switch_19 = 34;            //备用
	required bool  switch_20 = 35;            //空滤阻塞报警开关
	required bool  switch_21 = 36;            //制动灯开关
	required bool  switch_22 = 37;            //备用
	required bool  switch_23 = 38;            //驻车制动开关
	required bool  switch_24 = 39;            //水温报警开关
	required bool  switch_25 = 40;            //低气压1报警开关
	required bool  switch_26 = 41;            //备用
	required bool  switch_27 = 42;            //低气压2报警开关
	required bool  switch_28 = 43;            //地址开关1
	required bool  switch_29 = 44;            //地址开关2
	required bool  switch_30 = 45;            //后舱门开关
	required bool  switch_31 = 46;            //备用
	required bool  switch_32 = 47;            //备用

	required bool  switch_33 = 48;            //备用
	required bool  switch_34 = 49;            //备用
	required bool  switch_35 = 50;            //备用
	required bool  switch_36 = 51;            //备用
	required bool  switch_37 = 52;            //备用
	required bool  switch_38 = 53;            //备用
	required bool  switch_39 = 54;            //备用
	required bool  switch_40 = 55;            //备用
	required bool  switch_41 = 56;            //备用
	required bool  switch_42 = 57;            //备用
}
